### ISSUE TITLE
Screen Reader not adding text to buttons

### ISSUE THREAD

**Issue by Kashkin** [FrDH/jQuery.mmenu#738]
Using 6.02, the screen reader plugin doesn't add markup to submenu buttons or anywhere else.

Recreated here: https://codepen.io/Kashkin/pen/jmKgGv

**Comment by FrDH** [comment]
Hi,
Thanks for bringing this to my attention.
The are some screen-reader text in the demo (see screenshot), but I think there should be more.

I'll investigate this issue later on.
![schermafbeelding 2017-05-16 om 12 40 37](https://cloud.githubusercontent.com/assets/1387163/26102391/128a8472-3a35-11e7-8a93-32e9b0d8c125.png)

**Comment by FrDH** [comment]
Found the issue, should be fixed in the next release.

**Comment by Kashkin** [comment]
Thanks Fred, I'll keep my eye out for the next release.
